src/provider/bpmn/properties/CompensationProps.js

```diff
@@ -69,11 +69,11 @@ function WaitForCompletion(props) {
   };
 
   const setValue = (value) => {
-    commandStack.execute('properties-panel.update-businessobject', {
-      element: element,
-      businessObject: compensateEventDefinition,
+    commandStack.execute('element.updateModdleProperties', {
```

Comment:
How about if we use `modeling.updateModdleProperties` explicitly?

Reply:
We could but I wanted to keep the changes as simple as possible. Also in the case of multiple commands, we have to use the command stack as there is no modeling API.

Reply:
OK, I guess we can also do that for the spring cleaning.